eng/Versions.props

```diff
@@ -69,8 +69,11 @@
     <MicrosoftCodeAnalysisVisualBasicWorkspacesPackageVersion>4.10.0-1.24072.4</MicrosoftCodeAnalysisVisualBasicWorkspacesPackageVersion>
     <MicrosoftCodeAnalysisWorkspacesCommonPackageVersion>4.10.0-1.24072.4</MicrosoftCodeAnalysisWorkspacesCommonPackageVersion>
     <MicrosoftCodeAnalysisWorkspacesMSBuildPackageVersion>4.10.0-1.24072.4</MicrosoftCodeAnalysisWorkspacesMSBuildPackageVersion>
+    <MicrosoftSourceBuildIntermediateroslynPackageVersion>4.10.0-1.24072.4</MicrosoftSourceBuildIntermediateroslynPackageVersion>
     <MicrosoftVisualStudioLanguageServicesPackageVersion>4.10.0-1.24072.4</MicrosoftVisualStudioLanguageServicesPackageVersion>
+    <MicrosoftSourceBuildIntermediatearcadeVersion>8.0.0-beta.24081.5</MicrosoftSourceBuildIntermediatearcadeVersion>
     <MicrosoftDotNetXliffTasksPackageVersion>1.0.0-beta.23475.1</MicrosoftDotNetXliffTasksPackageVersion>
+    <MicrosoftSourceBuildIntermediatexlifftasksVersion>1.0.0-beta.23475.1</MicrosoftSourceBuildIntermediatexlifftasksVersion>
```

Comment:
I thought we didn't want unused properties to be here.

Reply:
Because the group is `Automated`, if these are missing, errors like `Missing version variable 'MicrosoftSourceBuildIntermediatearcadeVersion' in the 'Automated' property group` occur. The same thing happened in aspnetcore.